### DISCUSSION TITLE
Visibility of elastalert 2 plugin from the kibana frontend

### DISCUSSION THREAD

**Discussion by aerestrepo23** [jertel/elastalert2#995] [Q&A] [answered]
Hello

Just had a question regarding the elastalert 2 plugin.

Previously i was using the old elastalert plugin which was integrated with kibana 7.10.2. In this version i was able to visualize the installed elastalert plugin on the kibana front (shown in picture bellow).

![image](https://user-images.githubusercontent.com/117368491/199764097-025f531c-5daa-457d-80da-b626f817c8dc.png)

I was wondering if it's still possible to visualize the plugin when using elastalert 2 with kibana 8.4.1. I've managed to integrate it on my Kubernetes enviroment and alerts are running well, but having the visualization with the alerts on the frontend would certainly be helpful.

Thanks.

**Comment by nsano-rururu** [comment, marked as answer]
The kibana plugin is not official.
If you don't say thank you in the next repository issue, it won't reach the maintainer.
https://github.com/Karql/elastalert-kibana-plugin
